target/consensource-compose
# Add docker-compose down logic to run script

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

Add `docker-compose down` command when running the helper script with `--run`

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- `./docker-helper.sh --run` 
- `Ctrl + c` and verify that the containers are stopped and killed
